src/MSBuild.Sdk.SqlProj.Templates/templates/sqlproj/sqlproj.csproj

```diff
@@ -1,11 +1,17 @@
 <Project Sdk="MSBuild.Sdk.SqlProj/#{NBGV_NuGetPackageVersion}#">
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>netstandard2.1</TargetFramework>
         <SqlServerVersion>#{SqlServerVersion}</SqlServerVersion>
         <RunSqlCodeAnalysis>#{CodeAnalysis}</RunSqlCodeAnalysis>
         <!-- For additional properties that can be set here, please refer to https://github.com/rr-wfm/MSBuild.Sdk.SqlProj#model-properties -->
     </PropertyGroup>
 
+    <ItemGroup>
+        <!-- These packages adds additional code analysis rules -->
```

Comment:
Perhaps we should mention in the comment here that we recommend using these, but can be removed if desired?